bitmap-team-TeamName/index.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const fileReadHelper = require(`${__dirname}/lib/bmp-file-helper.js`);
+const bmpCon = require(`${__dirname}/model/bmp-constructor.js`);
+const makeBitMap = bmpCon.makeBitMap;
+
+
+if (!process.argv[2] || !process.argv[3]) {
+  console.error('Please enter two parameters: file name and transformation type (grayscale, invert, blue, green, red, yellow, purple)');
+  console.log('Format: node index <filename> <transformation type> Example: node index color-palette.bmp invert');
+  throw('error');
+}
+fileReadHelper.getBitMap(makeBitMap);
```

Comment:
@npisciotti1 @radenska @sneakyneeks2 @darms great job checking command line arguments and handling any errors if the proper filename/transformation type were not provided

additionally, great job creating and including a `readFile` helper module for reusability

side note:  no need to use `${__dirname}` in your require statements - simple relative paths are fine for this and have become an industry standard